token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -40,7 +47,7 @@ use {
 /// and `WithdrawWithheldTokensFromAccounts` instructions require corresponding zero-knowledge
 /// proof instructions.
 #[cfg(feature = "proof-program")]
-fn decode_proof_instruction<T: Pod>(
+pub(crate) fn decode_proof_instruction<T: Pod>(
```

Comment:
Yep, I mentioned it near the end of the PR description, but I think it would be more natural to do it in a separate follow-up PR with other plumbing necessary for the 1.16 zk stuff.

Reply:
Ah sorry about that! Retracting